### ISSUE TITLE
Can this be fixed within TMP? Strange behavior when using multi rail tran stations

### ISSUE THREAD

**Issue by VictorPhilipp** [krzychu124/Cities-Skylines-Traffic-Manager-President-Edition#157]
The pathfinder seems to take the way Cims have to walk from the rail to the exit of the station into account! I reproducibly observed that Cims try to switch to a train which arrives closest to the exit (for the multi rail train station). While the recently introduced path-finding costs for public transport transitions can indeed help to mitigate consequential negative effects to some extend, I still observe that Cims either do unnecessary switches or the usage of trains is not balanced (trains which arrive closer to the exit get more load).

Are there some variables or functionality to fix this within this mod? Or is this something Paradox has to fix within their code? (they never respond to forum posts clearly describing this issue though)

**Comment by VictorPhilipp** [comment]
Which station assets in particular exhibit the problem?

Most multi-track stations will have inbuilt pedestrian underpass or overpass - in which case this could be related to #144?

For stations that don't then at the very least you'd need roads both sides and also expect to see pedestrians walking across train tracks.

**Comment by VictorPhilipp** [comment]
Yes, I mean the inbuild pedestrian underpass or overpass. Those must have an effect otherwise I cannot explain why tracks more on the outside get less passengers in favor of trains who are more on the inside (even if trains on the inside are not the optimal route)

**Comment by aubergine10** [comment]
Was this issue fixed by introduction of "Prevent unecessary transfers" mod setting? Or is it something else?
